extensions/extension.go

```diff
@@ -115,3 +126,26 @@ func sprintf(args []object.Object) object.Object {
 	res := fmt.Sprintf(args[0].(object.String).Value, object.Unwrap(args[1:])...)
 	return object.String{Value: res}
 }
+
+func convertMap(m map[any]any) map[string]any {
+	result := make(map[string]any, len(m))
+	for key, value := range m {
+		if valueMap, ok := value.(map[any]any); ok {
+			value = convertMap(valueMap)
+		}
+		result[fmt.Sprint(key)] = value
+	}
+	return result
```

Comment:
Sad that this horror is needed

Considering just doing my own serialization instead

Reply:
removed it